### PR TITLE
Disable Full-Text Search for APG15

### PR DESCRIPTION
**Pull request by roshan0708**
### Description

Currently, APG15 includes basic full-text search features such as support for `CONTAINS`, `CREATE FULLTEXT INDEX`, and `DROP FULLTEXT INDEX` clauses. This commit disables all full-text search features for APG15, as full-text search is targeted for APG16.

### Issues Resolved

Task: BABEL-4987

### Test Scenarios Covered ###
* **Use case based -**

* **Boundary conditions -**

* **Arbitrary inputs -**

* **Negative test cases -**

* **Minor version upgrade tests -**

* **Major version upgrade tests -**

* **Performance tests -**

* **Tooling impact -**

* **Client tests -**

### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is under the terms of the Apache 2.0 and PostgreSQL licenses, and grant any person obtaining a copy of the contribution permission to relicense all or a portion of my contribution to the PostgreSQL License solely to contribute all or a portion of my contribution to the PostgreSQL open source project.

For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp [hunk 1]
```diff
@@ -909,20 +917,50 @@ antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitAlter_database(TSqlParser:
 	return visitChildren(ctx);
 }
 
+antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitFreetext_predicate(TSqlParser::Freetext_predicateContext *ctx)
+{
+	if (!pltsql_allow_fulltext_parser)
+    {
```
Comment (shalinilohia50) [inline contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp:923]: Nit: indentation seems off.

File: contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp [hunk 2]
```diff
@@ -909,20 +917,50 @@ antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitAlter_database(TSqlParser:
...
+                errmsg("Full Text Search is not currently supported in Babelfish.")));
+    }
+	handle(INSTR_UNSUPPORTED_TSQL_FREETEXT, "CONTAINS/FREETEXT predicate", &st_escape_hatch_fulltext, getLineAndPos(ctx));
+	return visitChildren(ctx);
+}
+
 antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitCreate_fulltext_index(TSqlParser::Create_fulltext_indexContext *ctx)
 {
+	if (!pltsql_allow_fulltext_parser)
+    {
```
Comment (shalinilohia50) [inline contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp]: Nit: indentation seems off.

**Review by shalinilohia50 — APPROVED** [review]

**Comment by bill-ramos-rmoswi** [comment]
Is it possible for customers to have started using the full-text features for the 3.5.0 release? If so, there is the possibility of breaking a customer's solution or impacting any upgrade if they started using any of the full-text features that slipped into 3.5.0. It would be best to just update the release notes rather than disabling the feature.

File: test/JDBC/upgrade/15_6/schedule [hunk 3]
```diff
@@ -223,8 +223,6 @@ format-dep
 forxml
 forxml-subquery
 fulltextserviceproperty
-FULLTEXT_INDEX
-fts-contains
```
Comment (shalinilohia50) [inline test/JDBC/upgrade/15_6/schedule:227]: If we are disabling this feature, let's ignore these tests in `jdbc_schedule` also. We would still have tests in `BABEL-UNSUPPORTED` to show that `FTS is not currently supported`.

**Pull request closed without merging**